code/modules/mob/living/carbon/carbon.dm

```diff
@@ -273,6 +273,25 @@
 
 	if(!item) return
 
+	if(istype(item, /obj/item/stack/throwing_knife))
+		var/obj/item/stack/throwing_knife/V = item
+		var/ROB_throwing_damage = src.stats.getStat(STAT_ROB)
+		V.throwforce = (35/(1+100/(ROB_throwing_damage))+10)
+		if(V.amount == 1)
+			drop_from_inventory(V)
```

Comment:
No need to drop if there is only one knife.

Reply:
It will not work if it won't be dropped from inv. Unless you want the knife to just do a funny animation in your hand?

Reply:
~~no, really, I removed the drop_from_inv thing and it didn't work during testing~~

Reply:
...yeah, it won't move from hands, despite being Move()'d. Funny.